### ISSUE TITLE
About Rl training

### ISSUE THREAD

**Issue by superhero-7** [aimagelab/meshed-memory-transformer#81]
Hi,dear author:
 Thanks your great job,I am rencently working on RL training in another task. I take your scst training code to reference, and I got a terrible result. My XE training is great,but after fintue by rl training for one epoch, the result crash heavily!!!
 Maybe my code is something wrong,but I also curious about whether there is any skill about rl training or not?Beacause I find that the loss beacome extremely unstable during rl training.
 If you have any advice, please tell me.I will appreciate very much if you can reply me as soon as possible!!!Thanks!!!!
